<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color: #3a3a3a;
	color: #fff;
	selection-background-color: #b78620;
	selection-color: #000;

}

/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QHeaderView-----*/
QHeaderView::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
    color: #fff;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


QTableCornerButton::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;

}

/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QPlainTExtEdit-----*/
QPlainTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QToolButton-----*/
QToolButton 
{
	background-color: transparent;
	color: #717072;
	padding: 5px;
	padding-left: 8px;
	padding-right: 8px;
	margin-left: 1px;
	border: 1px solid #808080;

}


QToolButton:hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;
	
}


QToolButton:pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #b78620;

}


QToolButton:checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #222;
}

/*-----QComboBox-----*/
QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
    border: 1px solid #000;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #b78620;
	color: #000;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b78620;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid; 

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;
}


/*-----QSpinBox &amp; QDateTimeEdit-----*/
QSpinBox,
QDateTimeEdit 
{
    background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	padding: 3px;
	padding-left: 5px;
    border-radius : 2px;

}


QSpinBox::up-button, 
QDateTimeEdit::up-button
{
	border-top-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-button:hover, 
QDateTimeEdit::up-button:hover
{
	background-color: #585858;

}


QSpinBox::up-button:pressed, 
QDateTimeEdit::up-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;
    height: 7px;

}


QSpinBox::down-button, 
QDateTimeEdit::down-button
{
	border-bottom-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-button:hover, 
QDateTimeEdit::down-button:hover
{
	background-color: #585858;

}


QSpinBox::down-button:pressed, 
QDateTimeEdit::down-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(://arrow-down.png);
    width: 7px;
    height: 7px;

}

/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #666666;
	border-radius: 5px;
    margin-top: 20px;

}


QGroupBox::title  
{
    background-color: transparent;
    color: #eee;
    subcontrol-origin: margin;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*PREDEFINED*/
#toolButton{
 border:none
}

#frTopDate{
 background-color:#808080
}

#frTop{
 background-color:#808080
}

#lblUser {
	background-color: transparent;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>94</y>
      <width>631</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::SubWindowView</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabsMovable">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>681</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="btnCustomer">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>85</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>CUSTOMER</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/1426825-sales/png/017-clerk-1.png</normaloff>recources/icons/1426825-sales/png/017-clerk-1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnPayment">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>253</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PAYMENT</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/1426825-sales/png/053-cash-register.png</normaloff>recources/icons/1426825-sales/png/053-cash-register.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnLedger">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>337</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>LEDGER</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/094-agenda.png</normaloff>recources/icons/957216-ui-interface/png/094-agenda.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnShs">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>169</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>SHS</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/1426825-sales/png/001-shop.png</normaloff>recources/icons/1426825-sales/png/001-shop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="b">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>422</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>SALES REPORT</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/037-open-book.png</normaloff>recources/icons/957216-ui-interface/png/037-open-book.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblDasboard">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>20</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
	font-family:Century Gothic;
	font-size:18px;
</string>
     </property>
     <property name="text">
      <string>DASHBOARD</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border:none</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/097-house.png</normaloff>recources/icons/957216-ui-interface/png/097-house.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnLedger_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>507</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>SETTINGS</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/091-settings.png</normaloff>recources/icons/957216-ui-interface/png/091-settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frTop">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblUser">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>21</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Consolas&quot;;
color: rgb(83, 255, 109);</string>
     </property>
     <property name="text">
      <string>Juan Dela Cruz</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>5</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/033-user.png</normaloff>recources/icons/957216-ui-interface/png/033-user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblUser_2">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>1</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(57, 0, 0);</string>
     </property>
     <property name="text">
      <string>WELCOME</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVleNo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Consolas&quot;;
color: rgb(255, 248, 190);</string>
     </property>
     <property name="text">
      <string>A1VLEA00001</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVleCode">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Consolas&quot;;
color: rgb(255, 248, 190);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frTopDate">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>10</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblUser_4">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>1</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(57, 0, 0);</string>
     </property>
     <property name="text">
      <string>TODAY IS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTime">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>29</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Consolas&quot;;
color: rgb(255, 248, 190);</string>
     </property>
     <property name="text">
      <string>--:--:--</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/025-hourglass.png</normaloff>recources/icons/957216-ui-interface/png/025-hourglass.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnShs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageShsSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCustomer</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageCustomerSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLedger</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageLedgerSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPayment</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>managePaymentSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageReportSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>manageCustomerSlot()</slot>
  <slot>manageShsSlot()</slot>
  <slot>clearMdi()</slot>
  <slot>manageLedgerSlot()</slot>
  <slot>managePaymentSlot()</slot>
  <slot>manageReportSlot()</slot>
 </slots>
</ui>
